apps/journeys/src/components/Conductor/Conductor.tsx

```diff
@@ -52,6 +52,10 @@ export function Conductor({ blocks }: ConductorProps): ReactElement {
     if (activeBlock != null && !activeBlock.locked) nextActiveBlock()
   }
 
+  const checkVideo = activeBlock?.children
+    .find((child) => child.__typename === 'CardBlock')
+    ?.children.find((child) => child.__typename !== 'VideoBlock')
```

Comment:
think you want to use `some`. since you want a boolean right?

Probably want something like videoBlockExists.

```suggestion
    ?.children.some((child) => child.__typename !== 'VideoBlock')
```